flowey/flowey_core/src/node.rs

```diff
@@ -1683,18 +1683,18 @@ pub mod steps {
             /// Adds a parameter to the step, specified as a key-value pair corresponding
             /// to the param name and value. For example the following code generates the following yaml:
             ///
-            /// ```rust
+            /// ```rust,ignore
```

Comment:
Do you actually want `ignore` here or do you want `no_run`?

Reply:
ignore is right - I haven't dug too deep into setting up a proper shared harness for writing these sorts of doctests yet (i.e: a dummy `ctx`)